circuits/cpp/barretenberg/cpp/src/barretenberg/proof_system/relations/ecc_vm/ecc_msm_relation.hpp

```diff
@@ -0,0 +1,102 @@
+  public:
+    using FF = FF_;
+    // 1 + polynomial degree of this relation
+    static constexpr size_t RELATION_LENGTH = 8;
+
+    // blarp... add
+    static constexpr size_t LEN_1 = 8; // arithmetic sub-relation
+    template <template <size_t...> typename AccumulatorTypesContainer>
+    using GetAccumulatorTypes = AccumulatorTypesContainer<LEN_1,
+                                                          LEN_1,
+                                                          LEN_1,
+                                                          LEN_1,
+                                                          LEN_1,
+                                                          LEN_1,
+                                                          LEN_1,
+                                                          LEN_1,
+                                                          LEN_1,
+                                                          LEN_1,
+                                                          LEN_1,
+                                                          LEN_1,
+                                                          LEN_1,
+                                                          LEN_1,
+                                                          LEN_1,
+                                                          LEN_1,
+                                                          LEN_1,
+                                                          LEN_1,
+                                                          LEN_1,
+                                                          LEN_1,
+                                                          LEN_1,
+                                                          LEN_1,
+                                                          LEN_1,
+                                                          LEN_1,
+                                                          LEN_1,
+                                                          LEN_1,
+                                                          LEN_1,
+                                                          LEN_1,
+                                                          LEN_1,
+                                                          LEN_1,
+                                                          LEN_1,
+                                                          LEN_1,
+                                                          LEN_1,
+                                                          LEN_1,
+                                                          LEN_1,
+                                                          LEN_1>;
+
+    /**
+     * @brief Expression for the StandardArithmetic gate.
+     * @details The relation is defined as C(extended_edges(X)...) =
+     *    (q_m * w_r * w_l) + (q_l * w_l) + (q_r * w_r) + (q_o * w_o) + q_c
+     *
+     * @param evals transformed to `evals + C(extended_edges(X)...)*scaling_factor`
```

Comment:
Old comments

Reply:
will remove as we have extended comment in .cpp